src/core/evaluator.js

```diff
@@ -419,28 +420,47 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         return Promise.resolve();
       }
 
-      var nativeImageDecoderSupport = this.options.nativeImageDecoderSupport;
+      const nativeImageDecoderSupport = forceDisableNativeImageDecoder ?
+        NativeImageDecoding.NONE : this.options.nativeImageDecoderSupport;
       // If there is no imageMask, create the PDFImage and a lot
       // of image processing can be done here.
-      var objId = 'img_' + this.idFactory.createObjId();
-      operatorList.addDependency(objId);
-      args = [objId, w, h];
+      const objId = 'img_' + this.idFactory.createObjId();
```

Comment:
This change is unrelated to the commit. Can you undo it? It might be better to consistently convert the whole function (or even file) from `var` to `let` / `const` in one commit or PR (e.g. for consistency, I would expect `w` and `h` to also be constant, at the very least).

Reply:
This change was reverted.